pos_default_invoice_active/static/src/js/pos_default_invoice_active.js

```diff
@@ -0,0 +1,25 @@
+/******************************************************************************
+ * Copyright (C) 2017-TODAY Camptocamp SA (<http://www.camptocamp.com>).
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ ******************************************************************************/
+
+odoo.define('pos_default_invoice_active.activate_invoicing', function (require) {
+    "use strict";
+
+  var pos_models = require('point_of_sale.models');
+  var pos_screens = require('point_of_sale.screens');
+
+  // add `iface_invoicing_active` to loaded pos config's fields
+  pos_models.load_fields("pos.config", "iface_invoicing_active");
+
+  pos_screens.PaymentScreenWidget.include({
+    show: function(){
+      this._super();
+      // activate invoicing
+      if (this.pos.config.iface_invoicing_active) {
+        this.$('.js_invoice').trigger('click');
```

Comment:
When you first go on the payment screen, invoice option is active. But if you didn't select any customer, you will be redirected on customer list to select one. After selecting a partner, go back on payment screen, you will see that invoice option is not enabled. Not sure if it's a bug or not, but it's confusing for a user to be asked to select a customer to be able to invoice the order, and then the user has to enable invoice feature again.

I think it's due to ```this.$('.js_invoice').trigger('click')```, when we go back on the payment screen, it toggles the feature.